src/lib/y2storage/callbacks/activate.rb

```diff
@@ -99,6 +118,61 @@ def luks_error(attempt)
 
         nil
       end
+
+      # @see IssuesCallback#error
+      def create_issue(message, what)
+        Issue.new(message, details: details(what))
+      end
+
+      # Issue details
+      #
+      # @param what [String]
+      # @return [String]
+      def details(what)
+        needle = /Cannot activate LVs in VG .* while PVs appear on duplicate devices/i
+        return what unless what.match?(needle)
+
+        [duplicated_pv_description, _("More details:"), what].join("\n\n")
```

Comment:
Weird. Although here we are joining those texts with "\n\n" they looks too close in the report

![installation_normal_mode](https://user-images.githubusercontent.com/1691872/143386042-5b0b3986-6c1a-480e-bff3-87ad5693b2a6.png)

Reply:
Good catch! Fixed.